README.md

```diff
@@ -24,9 +24,19 @@ mkdir build && cd build
 
 3. Building the library
 ```
-cmake --build . --config Release
+cmake --build . --config Release -j 4
```

Comment:
```suggestion
cmake --build . --config Release -j
```
Better to use `-j` without arguments or to be more explicit type `--parallel`, it will use all available cores. CMake documentation says:

```
       -j [<jobs>], --parallel [<jobs>]
              New in version 3.12.

              The maximum number of concurrent processes to use when building.  If <jobs> is omitted the native build tool's default number is used.

              The CMAKE_BUILD_PARALLEL_LEVEL environment variable, if set, specifies a default parallel level when this option is not given.

              Some native build tools always build in parallel.  The use of <jobs> value of 1 can be used to limit to a single job.
```